srcs/Mesh2D.cpp

```diff
@@ -0,0 +1,228 @@
+/**
+ * \file Mesh2D.cpp
+ * \brief Implementation of the required function to load a Mesh2D
+ * struct from file.
+ *
+ */
+
+#include <iostream>
+#include <gmsh.h>
+#include "Mesh2D.hpp"
+
+/**
```

Comment:
I think it is better for the code clarity to put the descriptions in the .hpp files

Reply:
Those are static functions, they are not defined in the .hpp (because they are not accessible from outside that .cpp).

Reply:
Oh yes sorry